data/pigui/pigui.lua

```diff
@@ -0,0 +1,80 @@
+local Format = import('Format')
+local Game = import('Game')
+local Space = import('Space')
+local Engine = import('Engine')
+local Event = import("Event")
+local ShipDef = import("ShipDef")
+local Vector = import("Vector")
+local Color = import("Color")
+local Lang = import("Lang")
+
+local lui = Lang.GetResource("ui-core");
+local lc = Lang.GetResource("core");
+local lec = Lang.GetResource("equipment-core");
```

Comment:
Same here, since you moved the "generating universe" string from core to ui-core.

Or is this needed in future PR? If so, then ignore comment.